x/ibc/03-connection/client/cli/tx.go

```diff
@@ -15,6 +15,10 @@ import (
 	host "github.com/cosmos/cosmos-sdk/x/ibc/24-host"
 )
 
+const (
+	flagProofEpoch, flagConsensusEpoch string = "proof-epoch", "consensus-epoch"
```

Comment:
I guess this is fine for now but I wonder if we should create a flags package within the ibc module